### PR TITLE
Added the week 3 development; used the translation api

### PR DESCRIPTION
**Pull request by MariferSalas**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslationServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (emilyzhang242) [inline portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java:32]: Nit: if it has to be named this feel free to ignore this comment, but please try to make this function name a little more descriptive.. What is the post request for?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -39,7 +40,7 @@ <h3>Social Profiles </h3>
         </p>
         
         <h3>Random Quotes</h3>
-      </div>
+        </div>
```
Comment (emilyzhang242) [inline portfolio/src/main/webapp/index.html]: Nit: adding the tab seems to make this unaligned with the opening div bracket. Please align.

**Review by emilyzhang242 — CHANGES_REQUESTED** [review]
Looks great!

File: portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java (hunk 1 above)
Comment (MariferSalas) [reply portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java:32]: I'll add a comment above the function to describe what it does, I put it that way to know that its related to the method POST

**Review by emilyzhang242 — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**